server/config-example.js

```diff
@@ -4,7 +4,7 @@ var config = {
   couchDbPort: '5984',
   couchDbUseSsl: false,
   couchDbChangesSince: 'now',
-  couchAdminUser: 'couchadmin',
+  couchAdminUser: 'hradmin',
```

Comment:
This is intentionally couchadmin, not hradmin.

There are two users that the script sets up:
1) _couchadmin_ - CouchDB admin.  This is a [CouchDB server administrator](http://docs.couchdb.org/en/2.0.0/config/auth.html?highlight=Admin%20party#server-administrators) which is a CouchDB super user.  This is used by hospitalrun-server-routes to do some user setup that a regular user can't do.
2) _hradmin_ - This is the default user to login to HospitalRun with.  This user is not a CouchDB server admin.